bucket/xray.json

```diff
@@ -0,0 +1,22 @@
+{
+	"version": "1.0.0",
+	"description": "Xray, Penetrates Everything. Also the best v2ray-core, with XTLS support. Fully compatible configuration.",
```

Comment:
Don't use the advertising words from the project itself. You may check out [an example](https://github.com/kidonng/sushi/blob/21e8c412c234324267ccdfc7f008296882a69387/bucket/xray.json#L3) of an appropriate description.